src/pybamm/input/parameters/lithium_ion/Ecker2015.py

```diff
@@ -368,15 +368,15 @@ def SEI_limited_dead_lithium_OKane2022(L_sei):
     Parameters
     ----------
     L_sei : :class:`pybamm.Symbol`
-        Total SEI thickness [m]
+        SEI thickness [m]
     Returns
     -------
     :class:`pybamm.Symbol`
         Dead lithium decay rate [s-1]
     """
 
     gamma_0 = pybamm.Parameter("Dead lithium decay constant [s-1]")
-    L_sei_0 = pybamm.Parameter("Initial SEI thickness [m]")
+    L_sei_0 = pybamm.Scalar(5e-9)
```

Comment:
Is this consistent if the initial SEI concentration changes, or should it be derived from that?